app/queries/allocations_and_unalocated_users_query.rb

```diff
@@ -17,7 +17,7 @@ def call
         (allocations.end_at > NOW() OR allocations.end_at IS NULL)'
       )
       .where(where)
-      .order('start_at NULLS LAST, end_at NULLS LAST, users.name')
+      .order('end_at NULLS LAST, start_at NULLS LAST, users.name')
```

Comment:
Can we get rid of this `start_at` ordering?

Reply:
No. If we remove the `start_at` ordering, the allocations without end will appear mixed with the unallocated ones.
![image](https://user-images.githubusercontent.com/25306897/181809584-974c87da-5cf6-4043-b5fd-1f430aac8c4c.png)